install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -12,262 +12,728 @@ toc::[]
+MySQL using NFS].
+
+For the cluster administrator, granting pods access to PVs involves:
+
+- knowing the group ID and/or user ID assigned to the actual storage,
+- understanding SELinux considerations, and
+- ensuring that these IDs are allowed in the range of legal IDs defined for the
+project and/or the SCC that matches the requirements of the pod.
+
+Group IDs, the user ID, and SELinux values are defined in the
+`*SecurityContext*` section in a pod definition. Group IDs are global to the pod
+and apply to all containers defined in the pod. User IDs can also be global, or
+specific to each container. Four sections control access to volumes:
+
+- link:#supplemental-groups[`*supplementalGroups*`]
+- link:#fsgroup[`*fsGroup*`]
+- link:#user-id[`*runAsUser*`]
+- link:#selinuxoptions[`*seLinuxOptions*`]
+
+[[sccs-defaults-allowed-ranges]]
+== SCCs, Defaults, and Allowed Ranges
+
+SCCs influence whether or not a pod is given a default user ID, `*fsGroup*` ID,
+supplemental group ID, and SELinux label. They also influence whether or not IDs
+supplied in the pod definition (or in the image) will be validated against a
+range of allowable IDs. If validation is required and fails, then the pod will
+also fail.
+
+SCCs define strategies, such as `*runAsUser*`, `*supplementalGroups*`, and
+`*fsGroup*`. These strategies are used to help decide if the pod is authorized.
+Strategy values set to *RunAsAny* are essentially stating that the pod can do
+what it wants regarding that strategy. Authorization is skipped for that
+strategy and no OpenShift default is produced based on that strategy. Therefore,
+IDs and SELinux labels in the resulting container are based on container
+defaults instead of OpenShift policies.
+
+For a quick summary of *RunAsAny*:
+
+- Any ID defined in the pod definition (or image) is allowed.
+- Absence of an ID in the pod definition (and in the image) results in the
+container assigning an ID, which is *root* (0) for Docker.
+- No SELinux labels are defined, so Docker will assign a unique (and generally not
+useful) label.
+
+For these reasons, SCCs with *RunAsAny* for ID-related strategies should be
+protected so that ordinary developers do not have access to the SCC. On the
+other hand, SCC strategies set to *MustRunAs* or *MustRunAsRange* trigger ID
+validation (for ID related strategies), and cause default values to be supplied
+by OpenShift to the container when those values are not supplied directly in the
+pod definition or image.
+
+SCCs may define the range of allowed IDs (user or groups). If range checking is
+required (for example, using *MustRunAs*) and the allowable range is not defined
+in the SCC, then the project determines the ID range. Therefore, projects
+support ranges of allowable ID. However, unlike SCCs, projects do not define
+strategies, such as `*runAsUser*`.
+
+Allowable ranges are interesting not only because they define the boundaries for
```

Comment:
Not sure I like the use of "interesting" here. Do you mean "important" or "helpful"? You may want to rephrase for clarity's sake.